lib/katello/plugin.rb

```diff
@@ -315,6 +315,8 @@
   Katello::PermissionCreator.new(self).define
   add_all_permissions_to_default_roles
 
+  add_permissions_to_default_roles 'System admin' => [:create_lifecycle_environments, :create_content_views]
```

Comment:
I *think* `add_permissions_to_default_roles` needs to go *inside* `Foreman::Plugin.register :katello do` to work properly

Reply:
nvm, I can't count.

Reply:
The whole file is one big block, I think it is inside.